src/html/logo.html

```diff
@@ -28,7 +28,7 @@ <h2 class="h2 weight-strong">Icon</h2>
   <h2 class="h2 weight-strong">Lockup</h2>
   <div class="figure-wrapper">
     <figure class="grid-block figure vertical figure-logo-lockup">
-      <img class="img" src="images/logo/runnable-logo/RGB/runnable-logo-rgb.svg" width="100%" alt"Lockup on Purple">
+      <img class="img" src="images/logo/runnable-logo/RGB/runnable-logo-rgb.svg" width="100%" height="140px" alt"Lockup on Purple">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7440805/25406041/3620f398-29ba-11e7-91a8-e7c410ebb8ca.png)
Also the lockups are still wide for me. They should both fit in one row (like the icons), at least at larger sizes.